src/extract-functions.js

```diff
@@ -1,11 +1,17 @@
 import isFunction from './isFunction';
 
-export default function (...args) {
-  let result = [];
-  for (let i = 0; i < args.length; i += 1) {
-    const arg = args[i];
-    if (isFunction(arg)) result.push(arg);
-    else if (Array.isArray(arg)) result = result.concat(arg.filter(isFunction));
+export default function extractFunctions() {
+  const fns = [];
```

Comment:
Why to rename to `fns`? I would say that `result` is much better to mark something that will be returned.